packages/cli/test/mocks/project.ts

```diff
@@ -223,14 +267,25 @@ export function useProject(project: Partial<Project> = defaultProject) {
     }
   );
   client.scenario.get(`/v8/projects/${project.id}/env`, (_req, res) => {
-    const target = _req.query.target;
-    if (typeof target === 'string') {
-      const targetEnvs = envs.filter(env => env.target.includes(target));
-      res.json({ envs: targetEnvs });
-      return;
+    const target: ProjectEnvTarget | undefined =
+      typeof _req.query.target === 'string'
```

Comment:
lets rename `_req` to `req`. The preceding `_` is meant to indicate "unused"

Reply:
on it! Gotta get out of that habit, I'm used to it meaning "private/protected" scope-wise 😅